tutorials/integration.md

```diff
@@ -343,213 +383,85 @@ for more information see here](https://medium.com/jeremy-keeshin/hello-world-for
 
 These are the steps to start the development:
 
-1. Install browserify so it will be available as a script.
-+	
-```bash
-sudo npm install -g browserify
-```
+1. Install browserify so it will be available as a script.	
+   ```bash
+   sudo npm install -g browserify
+   ```
 1. Create and change to a directory.
 1. Run this command to create the initial package definition file:
-+
-```bash
-npm init -y
-```
+   ```bash
+   npm init -y
+   ```
 1. Install the GSN package and its dependencies:
-+
-```bash
-npm install @opengsn/gsn@"^2.0.0" ethers
-```
+   ```bash
+   npm install @opengsn/gsn@"^2.0.3" ethers
+   ```
 1. Write your code in a file, for example `index.js`. You can use 
-[`require`](https://nodejs.org/en/knowledge/getting-started/what-is-require/) just as you 
-would with Node.js.
+   [`require`](https://nodejs.org/en/knowledge/getting-started/what-is-require/) just as you 
```

Comment:
please don't teach node. assume the user knows what require('module') means when creating a project.

Reply:
remove this. we're not teaching first-grade javascript. normal app contains many `require`s, so it look ridiculous to explain it.